kyverno/allow-host-ports.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: allow-host-ports
+  namespace: openebs
+  annotations:
+    policies.kyverno.io/category: Pod Security Standards (Privileged)
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      Access to host ports allows potential snooping of network traffic and should be
+      allowed, or at minimum restricted to a known list.      
+spec:
+  validationFailureAction: audit
+  background: true
+  rules:
+    - name: host-ports
+      match:
+        resources:
+          kinds:
+            - Pod
+      validate:
+        message: >-
+          Use of host ports is allowed. The fields spec.containers[*].ports[*].hostPort
+          and spec.initContainers[*].ports[*].hostPort must be empty.          
+        pattern:
+          spec:
+            =(hostPort):
```

Comment:
q: If we have hostnetwork enabled, do we still need to explicitly mention about hostports in the policy.?